docs/blog/2018-02-09-announcing-gatsby-manor-themes-for-gatsbyjs/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Announcing Gatsby Manor, themes for GatsbyJS 🎉🎊
+date: "2018-02-09"
+image: "gatsbymanor.jpg"
+author: "Steven Natera"
+---
+
+I am proud to announce [Gatsby Manor](https://www.gatsbymanor.com/), a gallery
+of themes for [GatsbyJS](https://www.gatsbyjs.org/). After 5 months of
+development, Gatsby Manor is now in public beta. If you are not familiar with
+GatsbyJS, use their [tutorial](https://www.gatsbyjs.org/tutorial/) to experience
+the delight of working with a [ReactJS](https://reactjs.org/) static site
+generator. When you are done learning, use one of our
+[themes](https://www.gatsbymanor.com/themes) to kickstart your GatsbyJS project.
```

Comment:
this blog audience is mostly already familiar with Gatsby, so you don't need to mention the tutorial